python/cuml/feature_extraction/_vectorizers.py

```diff
@@ -27,12 +29,11 @@
 
 def _get_non_alphanumeric_characters(docs):
     characters = docs.str.character_tokenize().unique()
-    non_alpha = characters.str.extract(r'([^\w])', expand=False).dropna()
-    return non_alpha.tolist() + ['\n']
+    non_alpha = characters.str.extract(r"([^\w])", expand=False).dropna()
```

Comment:
Meta question: I did a black run on this file and that added a lot of linting fixes, should I revert this back to make reviewing easier, or do we prefer black runs? 

Please, let me know what our protocol here is and I will change this accordingly.